core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/WorkspaceManagerApiClientProvider.scala

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.dsde.rawls.dataaccess.workspacemanager
+
+import bio.terra.workspace.api.WorkspaceApplicationApi
+import bio.terra.workspace.client.ApiClient
+
+/**
+ * Represents a way to get various workspace manager clients
+ */
+trait WorkspaceManagerApiClientProvider {
+  def getApiClient(accessToken: String): ApiClient
+
+  def getWorkspaceApplicationApi(accessToken: String): WorkspaceApplicationApi
+}
+
+class HttpWorkspaceManagerClientProvider(baseWorkspaceManagerUrl: String) extends WorkspaceManagerApiClientProvider {
```

Comment:
@cahrens I added this test to get coverage on the new WSM DAO enable app path

Reply:
@aherbst-broad looks like there are some test failures